tests/FSharp.Compiler.ComponentTests/Conformance/PatternMatching/NamedPatPairs/NamedPatPairs.fs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Conformance.PatternMatching
+
+open Xunit
+open FSharp.Test
+open FSharp.Test.Compiler
+
+module NamedPatPairs =
+    [<Theory; FileInlineData("NamedPatPairs01.fs")>]
+    let ``NamedPatPairs - NamedPatPairs01_fs - --test:ErrorRanges`` compilation =
```

Comment:
langVersion=9 should still error on the comma
langVersion=10/preview should allow both

The semicolon should for sure not be an error now, but maybe an info diagnostics with a message that it will become a warning in later versions (so that IDEs can do a codefix to react on it)

Reply:
Yeah. Im still working on it :). Will come back to this at some point